packages/graphql-language-service-utils/src/__tests__/Range-test.js

```diff
@@ -0,0 +1,73 @@
+import {expect} from 'chai';
+import {describe, it} from 'mocha';
+import {Range, Position, offsetToPosition, locToRange} from '../Range';
+
+const text = `query test {
+  name
+}`;
+const absRange = {start: 15, end: 18}; //position of 'name' attribute in the test query
+const offsetRangeStart = new Position(1, 2);
+const offsetRangeEnd = new Position(1, 5);
+
+describe('class Position', () => {
+  it('constructs Position object', () => {
+    const pos = new Position(3, 5);
+    expect(pos).to.not.be.undefined;
```

Comment:
Normally `to.not.be.undefined` wouldn't have any side effects, but I think it does due to "chai"'s dark magic. 😢